server/bin/pbench-backup-tarballs.py

```diff
@@ -230,38 +231,42 @@ def backup_to_s3(s3_obj, logger, controller_path, controller, tb, tar, resultnam
         # exist, or for other errors where we still want to backup locally.
         return Status.FAIL
 
-    logger.debug("Start S3 backup of {}.".format(tar))
+    logger.debug("Start S3 backup of {}.", tar)
     s3_resultname = os.path.join(controller, resultname)
 
     # Check if the result already present in s3 or not
     try:
-        obj = s3_obj.get_tarball_header(Bucket=s3_obj.bucket_name,
-                                        Key='{}'.format(s3_resultname))
-    except Exception:
+        tbh = s3_obj.get_tarball_header(Bucket=s3_obj.bucket_name,
+                                        Key=s3_resultname)
+    except NoSuchKey:
         s3_md5 = None
+    except Exception as e:
+        logger.error(str(e))
```

Comment:
Could we add an error message that gives some context, like: `logger.error("Error fetching tar ball header from S3: {}", e)`?